phase1/vsphere/README.md

```diff
@@ -75,40 +72,50 @@ If you do power it on, future clones of this VM will end up [getting the same IP
 echo -n > /etc/machine-id
 ```
 
+### Launch deployment environment
 
-### Download `kubernetes-anywhere`:
+#### Pull `cnastorage/kubernetes-anywhere:latest` docker image:
 
```

Comment:
Just feel all the subtitles under "Launch deployment environment" are too long, which makes this part a little bit not very neat...
They are more like content but not headings? Just wondering how does it look if making them into steps instead of "####" subtitles.